style.json

```diff
@@ -4219,10 +4142,6 @@
               "train_station_entrance",
               "waste_basket"
             ],
-            [
-              "has",
-              "name"
-            ],
```

Comment:
In this layer `poi-level-3`, the condition about "subclass" = "recycling" could also be removed.  
It was implemented in #101 + #102 to add interactivity specifically on recycling items.